src/main/scala/org/ergoplatform/nodeView/state/ErgoStateReader.scala

```diff
@@ -10,15 +10,31 @@ import scorex.util.ScorexLogging
 
 trait ErgoStateReader extends NodeViewComponent with ScorexLogging {
 
+  /**
+   * Root hash and height of AVL+ tree authenticating UTXO set
+   */
   def rootHash: ADDigest
```

Comment:
Please what the `height` in the description mean. This field represents only hash.

Reply:
Height of the tree as ScalaDoc says

Reply:
It is not clear what height has to do with this property, which is only a hash.

Reply:
renamed to rootDigest